ls8/cpu.c

```diff
@@ -2,6 +2,24 @@
 
 #define DATA_LEN 6
 
+
+/**
+ * Read value from address
+ */
+void cpu_ram_read(struct cpu *cpu, unsigned char address)
+{
+  return cpu->ram[address];
+}
+
+/**
+ * Write value to address
+ */
+void cpu_ram_write(struct cpu *cpu, unsigned char address, unsigned char *value)
```

Comment:
value should be number (although as we discussed it will be a char type) so you shouldn't need a pointer (asterisk) here or on line 19 as would maybe be the case with strings. It is of course possible to pass in `int *`s  but there's no reason to do so.